server_admin/topics/user-federation/ldap.adoc

```diff
@@ -125,6 +125,10 @@ FullName Mapper::
   This allows you to specify that the full name of the user, which is saved in some LDAP attribute (usually `cn` ) will be mapped to `firstName` and `lastname` attributes in the {project_name} database.
   Having `cn` to contain full name of user is a common case for some LDAP deployments.
 
+Hardcoded Attribute Mapper::
+  This allows you to add an hardcoded attribute value to each {project_name} user linked with LDAP. 
```

Comment:
1. Should be "a hardcoded" not "an hardcoded"
2. Try to always use a noun after this to make it more clear.  "This mapper... 
3. Also, try to avoid "Allows you to." which takes the focus off the user and puts it on the product. Try "This mapper adds a harcoded attribute..."